### PR TITLE
[CIT-366] Minimal monitoring stats for memory and disk

### PR DESCRIPTION
**Pull request by albinsuresh**
## Proposed changes

Limit the monitoring stats published for memory and disk to just the "used" percent values and ignore other values like "free", "reserved" etc.

## Types of changes

What types of changes does your code introduce to `thin-edge.io`?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (general improvements like code refactorings that doesn't explicitly fix a bug or add any new functionality)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
<br/>

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [ ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: configuration/contrib/collectd/tedge-collectd.conf [hunk 1]
```diff
@@ -1654,3 +1654,30 @@ LoadPlugin mqtt
 <Include "/etc/collectd/collectd.conf.d">
 	Filter "*.conf"
 </Include>
+
```
Comment (albinsuresh) [inline configuration/contrib/collectd/tedge-collectd.conf:1657]: For testing, add this snippet to the **end** of your existing `collectd.conf` file

**Review by PradeepKiruvale — APPROVED** [review]
LGTM and it collects only one stat per parameter is that right?

File: configuration/contrib/collectd/tedge-collectd.conf [hunk 2]
```diff
@@ -1654,3 +1654,30 @@ LoadPlugin mqtt
...
+LoadPlugin match_regex
+
+<Chain "PreCache">
+	<Rule "memory_free_only">
+		<Match "regex">
+			Plugin "memory"
+		</Match>
+		<Match "regex">
+			TypeInstance "used"
```
Comment (albinsuresh) [inline configuration/contrib/collectd/tedge-collectd.conf:1668]: With this change, we'll only be reporting the `used` percentage of `memory`.

**Comment by albinsuresh** [comment]
> LGTM and it collects only one stat per parameter is that right?

Yes. Only the percentage usage of cpu, memory and disk.

**Pull request merged**